lib/praxis/tasks/routes.rb

```diff
@@ -22,6 +21,10 @@
 
         method_name = method ? "#{method.owner.name}##{method.name}" : 'n/a'
 
+        if action.routes.empty?
+          raise "No routes defined for #{resource_definition.name}##{name}."
+        end
+
```

Comment:
I think rather than raise here, we should add a row for the action that just has no route information. 

Something like this if the routes are empty:
```ruby
rows << {
  resource: resource_definition.name,
  version: '',
  verb: '',
  path: '',
  action: name,
  implementation: method_name,
  name: '',
  primary: ''
}
```